src/antares/service/local_services/link_local.py

```diff
@@ -55,13 +52,6 @@ def create_link(
         """
         areas = dict(sorted({area_from: area_from, area_to: area_to}.items()))
```

Comment:
Minor: Line 53 and 55 can now be simplifed in `area_from, area_to =  sorted([area_from, area_to])`